test/clj_pgp/test/encryption.clj

```diff
@@ -105,6 +106,45 @@
               (is @error-occured? "A PGP error was simulated but not passed to the error handler."))))))))
 
 
+(def ^:private packet-header-byte
+  "
+  From right to left:
+    - Bits 1–0 are set to `00` to indicate a one-octet length packet.
+    - Bits 5–2 specify the packet's tag, which is `0001`.
+    - Bit 6 is `0` to indicate an 'old format' packet.
+    - Bit 7 is always `1` in a packet header.
+  See RFC 4880 section 4.2.
+  "
+  2r10000100)
+
+
+(def ^:private packet-length-byte
+  "
+  A packet's length doesn't include the first two header bytes.
+  This is an empty packet, so we're just using `0`
+  "
+  2r00000000)
```

Comment:
These are specific to the empty packet test below, so why are you defining them as top-level vars? That implies to me that they're used in multiple tests, but that's not the case. Making them regular let bindings in the test case would make it immediately obvious that they are only relevant to that test scope.

Reply:
Agreed.  I didn't like making them vars either, even private ones.  I was just trying to avoid cluttering the let binding with a bunch of inline comments.

I'll move them into the `let`.